src/monmove.cpp

```diff
@@ -88,6 +88,9 @@ bool monster::is_immune_field( const field_type_id &fid ) const
     if( fid == fd_insecticidal_gas ) {
         return !made_of( material_id( "iflesh" ) ) || has_flag( MF_INSECTICIDEPROOF );
     }
+    if( fid == fd_web ) {
+          return has_flag( MF_WEBWALK );
```

Comment:
```suggestion
        return has_flag( MF_WEBWALK );
```
Sigh.... I guessed on the indentation there and didn't get it right - should have copied it off the line above. Re astyle, etc, you should probably take a look at doc/CODE_STYLE.md and doc/DEVELOPER_TOOLING.md.

Reply:
I'll have a look at those, thank you. I tried to make sure my new code looked like everything around it but... obviously something wasn't quite right. Thanks very much for your input!